CHANGELOG.md

```diff
@@ -1,2 +1,23 @@
 
 ## Please see [Releases](https://github.com/SignalK/signalk-server-node/releases) for the release notes.
+
```

Comment:
As you see this file is not really used anymore and just points to the releases. The release notes in Releases are automatically generated from Pull Requests.